doc/SONiC_static_route_hdl.md

```diff
@@ -0,0 +1,370 @@
+nexthop             = 10.1.1.3
+ifname              = ""
+distance            = 0
+nexthop-vrf         = ""
+
+
+sonic(config)# ip route 10.1.1.1/24 Ethernet12 20
+
+Assumption is Ethernet12 is configured in Vrf-RED by user.
+STATIC_ROUTE table entries are updated for newly added route:-
+key                 = STATIC_ROUTE|default|10.1.1.1/24;
+vrf-name            = default
+prefix              = 10.1.1.1/24
+nexthop             = 10.1.1.3, 0.0.0.0
+ifname              = "", Ethernet12
+distance            = 0, 20
+nexthop-vrf         = "", Vrf-RED.
+````
+
+````
+
+sonic(config)# ip route vrf Vrf-RED 10.5.6.6/24 Ethernet0 10
+
+Assumption is Ethernet0 is in default VRF.
+A new STATIC_ROUTE table will be filled with following entries for Vrf-RED and prefix:-
+key                 = STATIC_ROUTE|Vrf-RED|10.5.6.6/24;
+vrf-name            = Vrf-RED
+prefix              = 10.5.6.6/24
+nexthop             = 0.0.0.0
+ifname              = Ethernet0
+distance            = 10
+nexthop-vrf         = default
+
+
+````
+#### IPv6 examples
+````
+sonic(config)# ipv6 route 2::/64 Ethernet16
+
+sonic(config)# ipv6 route 2001:FF21:1:1::/64 18:2:1::1 100
+````
+##### 3.6.2.2.2 ipv4 /ipv6 Command to delete a next-hop.
+
+The vrf is an optional parameter.
+
+```
+no ip route vrf <vrf-name> <prefix: A.B.C.D/mask> {[interface <interface-name>] | [<next-hop-ip>] | [<next-hop-ip> interface <interface-name>]}
+
+```
+
+```
+no ipv6 route vrf <vrf-name> <prefix: A.B.C.D/mask> {[interface <interface-name>] | [<next-hop-ip>] | [<next-hop-ip> interface <interface-name>]}
+```
+#### 3.6.2.3 Debug Commands
+#### 3.6.2.4 IS-CLI Compliance
+The following table maps SONiC CLI commands to corresponding IS-CLI commands. The compliance column identifies how the command comply to the IS-CLI syntax:
+
+- **IS-CLI drop-in replace**  \u2013 meaning that it follows exactly the format of a pre-existing IS-CLI command.
+- **IS-CLI-like**  \u2013 meaning that the exact format of the IS-CLI command could not be followed, but the command is similar to other commands for IS-CLI (e.g. IS-CLI may not offer the exact option, but the command can be positioned is a similar manner as others for the related feature).
+- **SONiC** - meaning that no IS-CLI-like command could be found, so the command is derived specifically for SONiC.
+
+|CLI Command|Compliance|IS-CLI Command (if applicable)| Link to the web site identifying the IS-CLI command (if applicable)|
+|:---:|:-----------:|:------------------:|-----------------------------------|
+|ip route |IS-CLI drop-in replace |Similar to OS-10 | |
+|ipv6 route| IS-CLI drop-in replace | Similar to OS-10 | |
+|no ip route |IS-CLI drop-in replace |Similar to OS-10 | |
+|no ipv6 route |IS-CLI drop-in replace |Similar to OS-10 | |
+
+
+### 3.6.3 REST API Support
+#### 3.6.3.1
+##### Get all support for both ARPs and Neighbors
+Various REST operations (POST/PUT/PATCH/GET/DELETE) will be supported for the static route configuration.

```

Comment:
Can multiple static routes be configured with single REST URI? Please add an example URI

Reply:
sure.

Reply:
done